packages/git/src/browser/git-navigable-list-widget.ts

```diff
@@ -101,30 +79,31 @@ export class GitBaseWidget<T extends { selected?: boolean }> extends VirtualWidg
 
     protected onAfterAttach(msg: Message): void {
         super.onAfterAttach(msg);
-        this.addKeyListener(this.node, Key.ARROW_LEFT, () => this.handleLeft());
-        this.addKeyListener(this.node, Key.ARROW_RIGHT, () => this.handleRight());
-        this.addKeyListener(this.node, Key.ARROW_UP, () => this.handleUp());
-        this.addKeyListener(this.node, Key.ARROW_DOWN, () => this.handleDown());
-        this.addKeyListener(this.node, Key.ENTER, () => this.handleEnter());
+        this.addKeyListener(this.node, Key.ARROW_LEFT, () => this.navigateLeft());
+        this.addKeyListener(this.node, Key.ARROW_RIGHT, () => this.navigateRight());
+        this.addKeyListener(this.node, Key.ARROW_UP, () => this.navigateUp());
+        this.addKeyListener(this.node, Key.ARROW_DOWN, () => this.navigateDown());
+        this.addKeyListener(this.node, Key.ENTER, () => this.handleListEnter());
+
     }
 
-    protected handleLeft(): void {
+    protected navigateLeft(): void {
         this.selectPreviousNode();
```

Comment:
I wanted to comment this directly on the function below but it seems I can ony comment on modified code so here it is. The condition in the if is wrong here 
```typescript
    protected selectPreviousNode() {
        const idx = this.indexOfSelected;
        if (idx > 0) {
            this.selectNode(this.gitNodes[idx - 1]);
        }
    }
```

Where when index is 0 it doesn't loop back to the end correctly. I think we can fix it in this PR.

Reply:
It was on purpose. Instead I avoid also the loop behaviour for the other direction because in history view it is more confusing than helpful. 
Do you think that it has a practical use?

Reply:
No your solution is perfect, I just wanted a consistent behaviour, either we loop from both sides or do nothing, and after thinking about it not looping seems to be the better solution indeed.